new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -0,0 +1,544 @@
+  private static final String OPERATOR_CHART_DIR =
+      "../kubernetes/charts/weblogic-operator";
+  private static final String OPERATOR_IMAGE =
+      //"weblogic-kubernetes-operator:latest";
+      "phx.ocir.io/weblogick8s/weblogic-kubernetes-operator:develop";
+
+  // mii constants
+  private static final String WDT_MODEL_FILE = "model1-wls.yaml";
+  private static final String MII_IMAGE_NAME_PREFIX = "mii-image-";
+  private static final String MII_IMAGE_TAG = "v1";
+
+  // domain constants
+  private static final String DOMAIN_VERSION = "v7";
+  private static final String API_VERSION = "weblogic.oracle/" + DOMAIN_VERSION;
+
+  private static HelmParams opHelmParams = null;
+  private static V1ServiceAccount serviceAccount = null;
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private String domainUID = "domain1";
+  private String miiImage = null;
+
+  /**
+   * Install Operator.
+   */
+  @BeforeAll
+  public static void initAll() {
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    opNamespace = createNamespace();
+    logger.info("Created a new namespace called {0}", opNamespace);
+
+    logger.info("Creating unique namespace for Domain");
+    domainNamespace = createNamespace();
+    logger.info("Created a new namespace called {0}", domainNamespace);
+
+    // Create a service account for the unique opNamespace
+    logger.info("Creating service account");
+    String serviceAccountName = createSA(opNamespace);
+    logger.info("Created service account: " + serviceAccountName);
+
+    // helm install parameters
+    opHelmParams = new HelmParams()
+        .releaseName(OPERATOR_RELEASE_NAME)
+        .namespace(opNamespace)
+        .chartDir(OPERATOR_CHART_DIR);
+
+    // Operator chart values to override
+    OperatorParams opParams =
+        new OperatorParams()
+            .helmParams(opHelmParams)
+            .image(OPERATOR_IMAGE)
+            .domainNamespaces(Arrays.asList(domainNamespace))
+            .serviceAccount(serviceAccountName);
+
+    // install Operator
+    logger.info("Installing Operator in namespace {0}", opNamespace);
+    assertThat(installOperator(opParams))
+        .as("Test installOperator returns true")
+        .withFailMessage("installOperator() did not return true")
+        .isTrue();
+    logger.info("Operator installed in namespace {0}", opNamespace);
+
+    // list helm releases
+    logger.info("List helm releases in namespace {0}", opNamespace);
+    assertThat(helmList(opHelmParams))
+        .as("Test helmList returns true")
+        .withFailMessage("helmList() did not return true")
+        .isTrue();
+
+    // check operator is running
+    logger.info("Check Operator pod is running in namespace {0}", opNamespace);
+    waitForCondition(operatorIsRunning(opNamespace));
+
+  }
+
+  @BeforeEach
+  public void init() {
+    // create unique namespace for domain and do operator upgrade?
```

Comment:
If this was left empty intentionally, why?  Why not delete the init method?

Reply:
Removed. One more test can be added for Operator upgrade.